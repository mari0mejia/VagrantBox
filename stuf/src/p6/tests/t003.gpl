int theend=2;

termination {
  print("This is one termination block. theend=" + theend);
  theend=12;
  print("Now theend=" + theend);
}
