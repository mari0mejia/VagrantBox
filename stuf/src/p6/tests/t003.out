gpl.cpp::main()
  input file(tests/t003.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int theend = 2
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().

User quit program.  Executing termination blocks
print[4]: This is one termination block. theend=2
print[6]: Now theend=12
